modules/nf-core/last/train/tests/main.nf.test.snap

```diff
@@ -0,0 +1,72 @@
+{
+    "sarscov2 - genome - contigs - stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "contigs",
+                            "single_end": false
+                        },
+                        "contigs..des.train:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "1": [
+                    "versions.yml:md5,c5578547acf9e77e1e8f6bf796e32ac2"
+                ],
+                "param_file": [
+                    [
+                        {
+                            "id": "contigs",
+                            "single_end": false
+                        },
+                        "contigs..des.train:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
This seems to be missing part of its name

Reply:
It is because the UNTAR module also returns a stub file and not the contents of the tarball, which does not cause failure because the files are not used by LAST_TRAIN when it runs as a stub. However, in that case, `    INDEX_NAME=\$(basename \$(ls $index/*.des) .des)` does not return the expected value, but `.des` instead, which is ugly but does not cause failure and is reproducible.  Shall we ignore it?  The index channel of LAST_TRAIN gets input stripped of its metadata because I was advised to do so when I created the module.

Reply:
Oh, is it just in the stub? Didn't notice that.
You could just define explictly `INDEX_NAME=STUB` or something then